PersonApi.Tests/V1/E2ETests/Steps/UpdatePersonStep.cs

```diff
@@ -27,7 +27,7 @@ public UpdatePersonStep(HttpClient httpClient) : base(httpClient)
         /// </summary>
         /// <param name="requestObject"></param>
         /// <returns></returns>
-        public async Task<HttpResponseMessage> CallApi(PersonRequestObject requestObject)
+        public async Task<HttpResponseMessage> CallApi(UpdatePersonRequestObject requestObject)
         {
             var token =
                 "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMTUwMTgxMTYwOTIwOTg2NzYxMTMiLCJlbWFpbCI6ImUyZS10ZXN0aW5nLWRldmVsb3BtZW50QGhhY2tuZXkuZ292LnVrIiwiaXNzIjoiSGFja25leSIsIm5hbWUiOiJUZXN0ZXIiLCJncm91cHMiOlsic2FtbC1hd3MtY29uc29sZS1tdGZoLWRldmVsb3BlciJdLCJpYXQiOjE2MjMwNTgyMzJ9.WffAEwWJlQorHGf-rIwxET8cJFK2yZg-kxNbtFctav4";
```

Comment:
Tokens should not be used here

Reply:
We can't do E2E without a token though. We can use a fake one... Tokens are nothing more than Base64 Encoded strings anyhow - are you worried about people knowing we're using OAuth2?

Reply:
agreed that the token is not a valid token so it's acceptable